src/index.js

```diff
@@ -1,16 +1,25 @@
 import './style.css';
-import Score from './module/Score.js';
-import addScore from './module/addScore.js';
-import { myScores, createScore } from './module/create.js';
+import getData from './module/get.js';
+import getResponse from './module/post.js';
 
-const storage = JSON.parse(localStorage.getItem('scores')) || [];
+const refreshBtn = document.getElementById('refresh');
+const SubmitBtn = document.querySelector('#submit');
+const nameInput = document.querySelector('#name');
+const scoreInput = document.querySelector('#score');
 
-for (let i = 0; i < storage.length; i += 1) {
-  const newScore = new Score();
-  newScore.name = storage[i].name;
-  newScore.score = storage[i].score;
-  myScores.push(newScore);
-  createScore(myScores[i], myScores[i].name, myScores[i].score);
-}
+refreshBtn.addEventListener('click', () => {
+  getData();
+});
 
-addScore();
\ No newline at end of file
+// add the click event to the submit btn
+
+SubmitBtn.addEventListener('click', () => {
+  const scoreInfo = {
+    user: nameInput.value,
+    score: scoreInput.value,
+  };
```

Comment:
- [ ] [Optional] It would be nice to validate the values of the inputs before a user posts their values to the API endpoint. This would ensure that users do not post blank entries. 👍